CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 ### Minor
 
 - Icon: Add new icon of `ads-stats` and `ads-overview` (#522)
```

Comment:
this is wrong, this update would include both of these PRs that landed, can we update?

Reply:
update and rebased. Thanks!